subscriptions.go

```diff
@@ -188,3 +190,9 @@ type SubscriptionNotes struct {
 	CustomerNotes         string   `xml:"customer_notes,omitempty"`
 	VATReverseChargeNotes string   `xml:"vat_reverse_charge_notes,omitempty"`
 }
+
+// CustomField is used for creating name - value pair.
+type CustomField struct {
```

Comment:
Since we plan to add `UnmarshalXML`, I suppose we will also have to add `MarshalXML` as well. Is that the correct understanding here? 

Also, I am still not sure why do we need map vs struct for custom_fields. Recurly does not enforce that name-value pair has to be unique. Is that the style and convention thing?

Reply:
They do enforce uniqueness on this field and it is a better API-user experience to get this data back in a map so it's easy to get the information you need directly instead of running a loop. I saw that you put up another commit - I will take a look at that later today!